test/test_checks.py

```diff
@@ -9,6 +21,29 @@ def test_get_noqa_lines(self):
         self.assertEqual(checker.get_noqa_lines(checker.get_file_contents()), [2])
 
 
+class TestFlake8Stdin(TestCase):
+    @expectedFailureIf(FLAKE8_VERSION > 2)
```

Comment:
As mentioned in the previous comment, please use an explicit check that `FLAKE8_VERSION` is equal to 3. We don't want to accidentally pass a test that should fail.